### DISCUSSION TITLE
关闭请求时后台输出请求地址

### DISCUSSION THREAD

**Discussion by Lier1210** [NervJS/taro#14015] [Q & A]
因为，项目安全有严格的要求，如何关闭框架自身在发起请求成功之后，【浏览器后台console.log】输出的请求地址，我之前在一篇帖子上看见过，但是想需要找不到了，还请解答一下

**Comment by Chen-jj** [comment]
是不是使用了 logInterceptor，一般来说不会 console.log 输出。可以截图一下调用栈看看

**Reply by Lier1210** [reply]
![image](https://github.com/NervJS/taro/assets/112168368/05d56146-f43a-4680-87cb-7ac7fb6a0e18)

**Reply by TheKonka** [reply]
你 build 有试过吗？ build 应该不会打印这些吧

**Reply by wzono** [reply]
Does that happen in a production environment?

**Reply by Lier1210** [reply]
生产环境中还没有试过，也不知道会不会有这种情况；
目前只是在开发开发阶段，还没上线使用的

**Reply by TheKonka** [reply]
不需要生产环境，你直接 run build:h5 然后打开产物看看会不会打印

**Comment by Lier1210** [comment]
向现在这样的，我之前看见过帖子，说是可以关闭&nbsp; 框架 内，不想显示的输出




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NervJS/taro"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年6月14日(星期三) 中午11:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NervJS/taro] 关闭请求时后台输出请求地址 (Discussion #14015)





 
是不是使用了 logInterceptor，一般来说不会 console.log 输出。可以截图一下调用栈看看

**Reply by TheKonka** [reply]
用邮件回复看不见图片
